src/main/charts/confluence/values.yaml

```diff
@@ -77,6 +77,14 @@ confluence:
     # -- The number of consecutive failures of the Confluence container readiness probe before the pod fails readiness checks
     failureThreshold: 30
 
+  accessLog:
+    # -- True if access logging should be enabled.
+    enabled: false
+    # -- The path within the Confluence container where the local-home volume should be mounted in order to capture access logs.
+    mountPath: "/opt/atlassian/confluence/logs"
+    # -- The subdirectory within the local-home volume where access logs should be stored.
+    localHomeSubPath: "logs"
```

Comment:
Does it mean, the access logs will be in `/opt/atlassian/confluence/logs/logs` directory?

Reply:
No. The `subPath` property of `volumeMount` refers to the subdirectory of the volume that should be mounted on `mountPath`. So the `logs` directory inside the `local-home` volume gets mounted on to the `/opt/atlassian/confluence/logs` directory in the container.